app/models/bid_status_presenter_factory.rb

```diff
@@ -100,23 +100,23 @@ def ineligible?
   end
 
   def work_in_progress?
```

Comment:
now that we call this `bidding_status`, `work_in_progress` no longer feels like it belongs here. thoughts?